src/main/java/com/mongodb/kafka/connect/source/MongoSourceConfig.java

```diff
@@ -130,6 +130,11 @@
             + "Defaults to the number of processors";
     private static final int COPY_EXISTING_MAX_THREADS_DEFAULT = Runtime.getRuntime().availableProcessors();
 
+    public static final String COPY_EXISTING_QUEUE_SIZE_CONFIG = "copy.existing.queue.size";
+    private static final String COPY_EXISTING_QUEUE_SIZE_DISPLAY = "Copy existing queue size";
+    private static final String COPY_EXISTING_QUEUE_SIZE_DOC =  "The max size of the queue to use when copying data from a namespace";
```

Comment:
Nit.  The queue is for all the namespaces

Reply:
Updated